internal/stack/_static/logstash.conf.tmpl

```diff
@@ -12,7 +12,7 @@ input {
 filter {
```

Comment:
In the `input` section of this configuration, those ssl certificates set there, should also be added to the agent ?

And I also see `ssl_enabled => false` , this should be enabled and the communication between logstash and agent should be with SSL.

This should be working in both Local stacks and Serverless stacks.